models.py

```diff
@@ -297,6 +298,25 @@ def from_payload(cls, payload: dict, dataset_id: str) -> "Resource":
         )
 
 
+@dataclass
+class CustomOrganization:
```

Comment:
Custom is too vague IMO. Maybe `TypedOrganization`, `AnnotatedOrganization`, or something else that conveys we're adding info to the organization entity, vs doing some other form of "custom" stuff.

We might want to pick a name for our "org (and possibly others) API" and use it here, as I suspect we'll need a way to refer to it anyway.
Not sure how to call it though? "Universe API", "Extension API", "Annex API", "Annotation API", ... ? Maybe even "Custom API" (full circle :v:) ?

Reply:
I like "Universe API" but it won't work well here: universe organisations are both in the topic and the `ecospheres-universe` API... Extension is cool too, but ExtendedOrganization won't work well for my base object, since it holds only extra attributes. Naming is hard :p

I went full literal / descriptive: `EcospheresUniverseOrganization`.

Reply:
Quite a mouthful, but better than something too generic. I think I'll nickname it Ecoverse :)